pkg/hubble/observer/local_observer.go

```diff
@@ -265,7 +265,7 @@ func (s *LocalObserverServer) GetFlows(
 		return err
 	}
 
-	flowsReader, err := newFlowsReader(ringReader, req, log, whitelist, blacklist)
+	flowsReader, err := newEventsReader(ringReader, req, log, whitelist, blacklist)
```

Comment:
Would it be worth renaming the variable to `eventsReader` as well then?

Reply:
Good catch! Will do, missed that one!

Reply:
Fixed; also renamed the label to `nextEvent` while at it: https://github.com/cilium/cilium/compare/741ddd7e581b7d674e23f3dc1064266d349de20c..a8c28ce0867d34d09da9cc9410845e54a6cb92ee